src/http_server.cc

```diff
@@ -67,6 +67,44 @@ namespace triton { namespace server {
     }                                                 \
   } while (false)
 
+#define HTTP_RESPOND_WITH_CODE(REQ, CODE, MSG)    \
+  do {                                            \
+    EVBufferAddErrorJson((REQ)->buffer_out, MSG); \
+    evhtp_send_reply((REQ), CODE);                \
+    return;                                       \
+  } while (false)
+
+namespace {
```

Comment:
I will swap this block to be with the rest of the helpers and move the HTTPServer Start/Stop and HTTPMetrics server to be down after the helpers with the rest of the server code in a quick follow-up PR. 

Doing so in this PR introduces a lot of noise to the amount of actual changes.